public/locales/en/translation.json

```diff
@@ -19,10 +19,11 @@
     "about": "About",
     "support": "Support",
     "english": "English",
-    "spanish": "Spanish",
-    "german": "German",
+    "spanish": "Español",
+    "german": "Deutsch",
```

Comment:
This being the EN Translation file, shouldn't we leave the Deutsch as German, it's English definition?

Reply:
I think its more user friendly if its in the same Language the user is looking for. For an more extreme example, imagine visiting an chinese website and looking for the english button but you dont know what english is in chinese. What do you think?